testng-core/src/main/java/org/testng/xml/LaunchSuite.java

```diff
@@ -272,7 +251,7 @@ static class ClassesAndMethodsSuite extends CustomizedSuite {
         final Map<String, String> parameters,
         final String annotationType,
```

Comment:
Could be removed too

Reply:
I want to revisit a bunch of classes in this `org.testng.xml` package and perhaps deprecate them or remove them directly based on my analysis. I will do this in a separate PR.